lib/instrumentation/langchain/tools.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const { LangChainTool } = require('../../llm-events/langchain')
+const common = require('./common')
+const {
+  AI: { LANGCHAIN }
+} = require('../../metrics/names')
+const runId = Symbol('runId')
+const { DESTINATIONS } = require('../../config/attribute-filter')
+
+module.exports = function initialize(shim, tools) {
+  const { agent, pkgVersion } = shim
+
+  if (common.shouldSkipInstrumentation(agent.config)) {
+    shim.logger.debug(
+      'langchain instrumentation is disabled.  To enable set `config.ai_monitoring.enabled` to true'
```

Comment:
Do we omit noting the feature flag here because we intend to remove it?

Reply:
yea the feature flag will be removed when we're ready for this instrumentation to be used so the message omits that indicator